.github/actions/test_conformance_suite/action.yml

```diff
@@ -0,0 +1,58 @@
+name: Test Conformance Suite
```

Comment:
arbitrary line:

Testing for this PR should include making a conformance suite fail and confirming error reporting continues to work properly in our CI.